frontend/src/templates/Field/Email/EmailFieldInput.tsx

```diff
@@ -24,12 +25,13 @@ export interface EmailFieldInputProps {
 
 export const EmailFieldInput = ({
   schema,
+  disableRequiredValidation,
   handleInputChange,
   inputProps = {},
 }: EmailFieldInputProps): JSX.Element => {
   const validationRules = useMemo(
-    () => createEmailValidationRules(schema),
-    [schema],
+    () => createEmailValidationRules(schema, disableRequiredValidation),
```

Comment:
Do we enforce the required validation upon the final submission?

Reply:
I dont think there's a concept of a final submission for MRF